.vib/minio/cypress/cypress/integration/minio_test_suite.js

```diff
@@ -0,0 +1,59 @@
+/// <reference types="cypress" />
+import {
+  random,
+} from './utils'
+
+it('allows user to log in and log out', () => {
+  cy.login();
+  cy.get('[data-testid="ErrorOutlineIcon"]').should('not.exist');
+  cy.get("#logout").scrollIntoView().should('be.visible').click();
+})
+
+it('allows creating a bucket and file upload', () => {  
+  cy.login();
+  cy.visit('add-bucket');
+  cy.fixture('testdata').then((td) => {
+    cy.get('#bucket-name').should('be.visible').type(`${td.bucketTitle}.${random}`);
+  })
+  cy.contains('button[type="submit"]','Create Bucket').click();
+  cy.contains('#upload-main','Upload').should('be.visible').click();
+  cy.contains('span','Upload File').should('be.visible').click();
+  cy.get('div#object-list-wrapper > input[type="file"]').selectFile('cypress/fixtures/example.json',
+  {force:true});
+})
+
+it('allows creating a user', () => {
+  cy.login();
+  cy.visit('identity/users');
+  cy.contains('Create User');
+  cy.get('[aria-label="Create User"]').should('be.visible').click();
+  cy.fixture('testdata').then((td) => {
+    cy.get('#accesskey-input').should('be.visible').type(`${td.testAccessKey}.${random}`);
+    cy.get('#standard-multiline-static').should('be.visible').type(`${td.testSecretKey}.${random}`);
+  })
+  cy.contains('button[type="submit"]','Save').should('be.visible').click();
+  cy.fixture('testdata').then((td) => {
+    cy.contains(td.testAccessKey).should('be.visible');
+    cy.get('#accesskey-input').should('not.exist');
+  })
+})
+
+it('allows creating a group', () => {
+  cy.login();
+  cy.visit('identity/groups');
+  cy.get('[aria-label="Create Group"]').should('be.visible').click();
+  cy.fixture('testdata').then((td) => {
+    cy.get('#group-name').should('be.visible').type(`${td.testGroupName}.${random}`);
+    cy.contains('button[type="submit"]','Save').click();
+    cy.contains(td.testGroupName);
```

Comment:
I think this is also somewhat flaky. If the form fails to save, it would still match because `#group-name` will have the value.

Reply:
I've added an additional assertion that checks the disappearance of the modal window. Check line 57.